templates/twig/basestyle.twig

```diff
@@ -24,10 +24,11 @@
         <p>Drops of rain could be heard hitting the pane, which made him feel quite sad. "How about if I sleep a little bit longer and forget all this <strong>nonsense</strong>", he thought, but that was something he was unable to do because he was used to sleeping on his right, and in his present state couldn't get into that position. However hard he threw himself onto his right, he always rolled back to where he was.</p>
         <p>He must have tried it a hundred times, shut his eyes so that he wouldn't have to look at the floundering legs, and only stopped when he began to feel a mild, dull pain there that he had never felt before.</p>
         <hr>
-        <div>
+        <div class="themeDefault centerMaxWidthContainer">
           <h2>Default Theme</h2>
-          <p>Not using theme's classes allows you to style a group of content with a default background and default text color.</p>
+          <p>Using the themeDefault class allows you to style a group of content with a default background and default text color.</p>
            <p>
+            <a href="#">Regular link</a><br><br>
```

Comment:
When using an anchor tag that navigates nowhere, perhaps it would be better instead of `#`, to use one of the following in the `href`:
- `#!` - reference to an id with a name of `!`
- `javascript:void(0)`

in order to prevent bouncing to the top of the page once the link has been clicked.

Reply:
Good point, will change.